.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         'simple-import-sort/exports': 'error',
         'no-unused-vars': 'off',
         'unused-imports/no-unused-imports': 'error',
+        'no-case-declarations': 'off',
```

Comment:
Lets not turn this off. We can fix this by use braces to ensure proper scope in the `case`

Reply:
Will update

Reply:
Have updated. Will leave resolution to you as the person confirming acceptance of the change.